### PR TITLE
hostapd: update to 2023-03-29

### PR DESCRIPTION
**Pull request by PolynomialDivision**
Upstream status:
http://lists.infradead.org/pipermail/hostap/2023-March/041450.html

Discussion about the nl80211 patch:
http://lists.infradead.org/pipermail/hostap/2023-March/041451.html

Edit:
Added a patch adding s8 to libnl-tiny:
http://lists.openwrt.org/pipermail/openwrt-devel/2023-March/040692.html
If that is accepted we can remove `180-nl80211-fix-compiling.patch`.

### CODE REVIEW

**Comment by Gingernut1978** [comment]
Running fine on my mediatek/filogic device

tested by andrewsimz@gmail.com

File: package/network/services/hostapd/patches/180-nl80211-fix-compiling.patch [hunk 1]
```diff
@@ -0,0 +1,48 @@
+From 319bae5ecf872355765fdb55baa169697f00d849 Mon Sep 17 00:00:00 2001
+From: Nick Hainke <vincent@systemli.org>
+Date: Tue, 14 Mar 2023 22:21:01 +0100
+Subject: [PATCH] nl80211: fix compiling
+
+This patch will not be sent upstream. It is just a workaround to fix
+following error:
```
Comment (hauke) [inline package/network/services/hostapd/patches/180-nl80211-fix-compiling.patch:7]: libnl-tiny was updated, you can remove this patch now.

**Comment by PolynomialDivision** [comment]
I think I will update it again to the latest version.

**Comment by PolynomialDivision** [comment]
@hauke What about merging this to get a recent hostapd version for the new release?

**Comment by pktpls** [comment]
Have been testing this with our Freifunk setup on Fritzbox 4040 (ipq40xx) and EAP225-Outdoor (ath79), it's a mix of 802.11s, WPA3 OWE, and WPA3 PSK - has been working fine from what I can tell. :+1:

**Comment by PolynomialDivision** [comment]
@Gingernut1978 Do you want to test again after the update? Than I can also add you as tested by.

**Comment by Gingernut1978** [comment]
`hostapd-common
2023-03-29-bb945b98-1.2`

Been running it for the last three days. No issues to report.

Great job.

**Comment by PolynomialDivision** [comment]
Added you as Tested by. Thanks a lot!

**Comment by PolynomialDivision** [comment]
@blocktrron Tested now over a longer time. Works flawlessly.

**Comment by Neustradamus** [comment]
@PolynomialDivision, @blocktrron, @dhewg: There is now an official build: 2.11 (2024-06-20) that I have requested:
- https://lists.infradead.org/pipermail/hostap/2024-July/042847.html

It is possible to update?

**Pull request merged**